src/LoopMachine.php

```diff
@@ -118,4 +119,27 @@ protected function _createStateInstance($value)
     {
         return new State($value);
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    public function canTransitionTo(ValueAwareInterface $state)
```

Comment:
Ah my bad! I meant to remove that after we settled on extending `FiniteStateMachineInterface`